readthedocs/projects/models.py

```diff
@@ -213,10 +213,24 @@ class Project(models.Model):
         ),
     )
 
+    # External versions
     external_builds_enabled = models.BooleanField(
         _('Build pull requests for this project'),
         default=False,
-        help_text=_('More information in <a href="https://docs.readthedocs.io/en/latest/guides/autobuild-docs-for-pull-requests.html">our docs</a>')  # noqa
+        help_text=_('More information in <a href="https://docs.readthedocs.io/page/guides/autobuild-docs-for-pull-requests.html">our docs</a>')  # noqa
+    )
+    external_builds_privacy_level = models.CharField(
+        _('Privacy level of builds from pull requests'),
+        max_length=20,
+        # TODO: remove after migration
+        null=True,
+        choices=constants.PRIVACY_CHOICES,
+        # Force it to private so current projects in .com
+        # use this value.
+        default=constants.PRIVATE,
+        help_text=_(
+            'Should builds from pull requests be public?',
+        ),
```

Comment:
This field is not a checkbox, so I think the help text shouldn't be a yes/no question.

Reply:
We are using something similar for the project privacy level.

Reply:
Yea, this should probably link to the privacy level docs, similar to the PR build.

Reply:
Still needs a link ^^

Reply:
do you mean linking to the section? Because we already link to the page just above

![image](https://user-images.githubusercontent.com/4975310/123185958-b2656000-d45c-11eb-9f88-70fc28872b49.png)

Reply:
ah, didn't see that 👍